client/src/components/forms/form-wrapper.component.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC, ReactElement } from 'react';
 
 interface FormWrapperProps {
-  children: ReactElement;
+  children: ReactElement | string;
```

Comment:
Remove this addition as there is little point to a `FormWrapper` component just showing a string, that is not a form. Consider the purpose of the component when writing your tests. Tests are meant to give you confidence the component can do what it was built for. This is not testing that.